base/applications/regedit/framewnd.c

```diff
@@ -1400,7 +1400,7 @@ LRESULT CALLBACK FrameWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lPa
             return DefWindowProcW(hWnd, message, wParam, lParam);
         break;
     case WM_ACTIVATE:
-        if (LOWORD(hWnd) && g_pChildWnd)
+        if (LOWORD(wParam) && g_pChildWnd)
```

Comment:
Maybe make it more explicit:
```suggestion
        if (LOWORD(wParam) != WA_INACTIVE && g_pChildWnd)
```
Like here:
- https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/winhlp32/winhelp.c;hb=965230ded43ddc1a16f3d0e7f293e1548f4bcaf4#l974
- https://git.reactos.org/?p=reactos.git;a=blob;f=base/system/rundll32/rundll32.c;hb=965230ded43ddc1a16f3d0e7f293e1548f4bcaf4#l342